examples/loader-python-to-png/src/index.md

```diff
@@ -31,14 +31,16 @@ To run this data loader, you’ll need python3 and the geopandas, matplotlib, io
 - `$ python3 -m venv .venv`
 - `$ source .venv/bin/activate`
 
+Then install the required modules:
+
+- `$ pip install -r requirements.txt`
+
 </div>
 
-The above data loader lives in `data/birth-statistics.png.py`, so we can load the data using `data/birth-statistics.png`. Access the PNG in a markdown page using FileAttachment:
+The above data loader lives in `data/birth-statistics.png.py`, so we can access the image using `data/birth-statistics.png`:
 
-```js echo
-const birthRateMap = FileAttachment("data/birth-statistics.png").image();
+```html run=false
+<img src="data/birth-statistics.png" style="max-width: 500px;">
```

Comment:
The code shows `max-width` here, but we’re not using it below.

Also, would prefer to show the code below the output rather than above, like we do for fenced code blocks that echo.